### PR TITLE
Sentence structure and wording improvements

### PR DESCRIPTION
**Pull request by lewislarsen**
I aimed to streamline the wording across strings to make things a little bit more clearer. 

Feel free to discuss the changes with me if you have any concerns or disagreements with my changes.

### CODE REVIEW

**Comment by yuliu** [comment]
Looks good to me.

A side note on the ending period/dot: it appears that current 1.8's theme should do with (most of) them.

.. and perhaps it might be better to create an issue to track the wording improvement.

File: inc/languages/english/member.lang.php [hunk 1]
```diff
@@ -195,48 +195,48 @@
...
-$l['redirect_lostpwsent'] = "Thank you, all accounts pertaining to that email address have now been sent an email with details on how to reset the passwords.<br /><br />You will now be taken to the forums' index.";
-$l['redirect_activationresent'] = "Your activation e-mail has been resent.";
-$l['redirect_passwordreset'] = "Thank you, the password for your account has been reset. The new randomly generated password has been emailed to the email address in your account.";
-$l['redirect_memberrated'] = "The member has successfully been rated.";
-$l['redirect_registered_passwordsent'] = "A random password has been generated and sent to your email address. Before you can login on these forums, you will need to check your email for this password.";
-$l['redirect_validated'] = "Thank you, your account has been validated.<br />You will now be taken to the forums.";
+$l['redirect_lostpwsent'] = "Thank you, all accounts pertaining to that email address have now been sent an email with details on how to reset their passwords.<br /><br />You will now be taken to the forum index.";
+$l['redirect_activationresent'] = "Your activation email has been resent to the email address providied.";
```
Comment (effone) [inline inc/languages/english/member.lang.php]: 'providied` is that so?

**Comment by Ben-MyBB** [comment]
Thanks Lewis, does anyone else have objections here? @mybb/developers 

@LewisLarsen Could you make an issue to track these for when this (hopefully) gets merged?

**Comment by euantorano** [comment]
Only issue I see is that one pointed out above regarding spelling. Once that’s fixed, I’m happy to merge.

File: inc/languages/english/member.lang.php (hunk 1 above)
Comment (euantorano) [inline inc/languages/english/member.lang.php]:
```suggestion
$l['redirect_activationresent'] = "Your activation email has been resent to the email address provided.";
```

**Pull request merged**